pkg/cli/cvd.go

```diff
@@ -412,3 +450,67 @@ func verifyCVDHostPackageTar(dir string) error {
 	}
 	return nil
 }
+
+type LocalArtifactsSrcs struct {
+	Images            map[string]string
+	Bootloader        string
+	CVDHostPackageTar string
+}
+
+func (s *LocalArtifactsSrcs) validate() error {
```

Comment:
OOC, would we need to validate if essential CF images exist or not? For example, checking whether `vbmeta.img` is defined in `LocalArtifactsSrcs` or not.

Reply:
good point, however I couldn't find any external documentation listing the required partition image for a cuttlefish device to give the user more context here, let's keep it flexible for now, we can always add this sort of validation later on.

Reply:
Okay, I also agree with you. We don't need this kind of validation immediately. If there's no rule on here, then it's fine for me :)